cmake-build-debug/tests/test_builder.h

```diff
@@ -0,0 +1,304 @@
+#ifndef TEST_BUILDER_H
```

Comment:
probably don't want this file checked in?

Reply:
Removed.